diff --git a/mkdocs.py b/mkdocs.py
index 25cb55e2..422f3603 100755
--- a/mkdocs.py
+++ b/mkdocs.py
@@ -10,17 +10,12 @@ root_dir = os.path.abspath(os.path.dirname(__file__))
 docs_dir = os.path.join(root_dir, 'docs')
 html_dir = os.path.join(root_dir, 'html')
 
-local = not '--deploy' in sys.argv
+local = False
 preview = '-p' in sys.argv
 
-if local:
-    base_url = 'file://%s/' % os.path.normpath(os.path.join(os.getcwd(), html_dir))
-    suffix = '.html'
-    index = 'index.html'
-else:
-    base_url = 'http://www.django-rest-framework.org'
-    suffix = ''
-    index = ''
+base_url = '/djangorestframework/2.4.8/'
+suffix = '.html'
+index = 'index.html'
 
 
 main_header = '<li class="main"><a href="#{{ anchor }}">{{ title }}</a></li>'
@@ -163,11 +158,7 @@ for (dirpath, dirnames, filenames) in os.walk(docs_dir):
         output = output.replace('{{ page_id }}', filename[:-3])
         output = output.replace('{{ canonical_url }}', canonical_url)
 
-        if filename =='index.md':
-            output = output.replace('{{ ad_block }}', """<hr/>
-              <script type="text/javascript" src="//cdn.fusionads.net/fusion.js?zoneid=1332&serve=C6SDP2Y&placement=djangorestframework" id="_fusionads_js"></script>""")
-        else:
-            output = output.replace('{{ ad_block }}', '')
+        output = output.replace('{{ ad_block }}', '')
 
         if prev_url:
             output = output.replace('{{ prev_url }}', prev_url)
